easybuild/easyconfigs/i/IGoR/IGoR-1.4.2-GCC-11.3.0.eb

```
@@ -0,0 +1,52 @@
+# Contribution from Imperial College Loncon / UK
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'IGoR'
+version = '1.4.2'
+
+homepage = 'https://github.com/statbiophys/IGoR'
+description = """
+This repository contains all sources and models useful to infer V(D)J 
+recombination related processes for TCR or BCR sequencing data using IGoR.
+This is the fork from the Statbiophys GitHub page!"""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
```

since you're depending on `FlexiBLAS`, we should probably be using the new `gfbf` toolchain (`GCC` + `FlexiBLAS` + (serial) `FFTW`)
```suggestion
toolchain = {'name': 'gfbf', 'version': '2022a'}
```

---

Using `EasyBuild 4.5.5` I get when switching to the `gfbf` toolchain:
```
ERROR: Missing dependencies: Autotools/20220317-gfbf-2022a, M4/1.4.19-gfbf-2022a, binutils/2.38-gfbf-2022a, GSL/2.7-gfbf-2022a, jemalloc/5.3.0-gfbf-2022a, gfbf/2022a (no easyconfig file or existing module found)
```
Clearly I am missing something here. Also, I could not find an EC for `gfbf/2022a`

---

@sassy-crick update to EasyBuild 4.5.6 :)

---

Using `EasyBuild 4.5.5` I get when switching to the `gfbf` toolchain:
```
ERROR: Missing dependencies: Autotools/20220317-gfbf-2022a, M4/1.4.19-gfbf-2022a, binutils/2.38-gfbf-2022a, GSL/2.7-gfbf-2022a, jemalloc/5.3.0-gfbf-2022a, gfbf/2022a (no easyconfig file or existing module found)
```
Clearly I am missing something here. Also, I could not find an EC for `gfbf/2022a`
@jfgrimm I was not aware that EasyBuild 4.6.0 is out. :-)

---

@jfgrimm I was not aware that EasyBuild 4.6.0 is out. :-)